src-tauri/tauri.conf.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "../node_modules/@tauri-apps/cli/schema.json",
   "build": {
-    "beforeBuildCommand": "",
-    "beforeDevCommand": "",
-    "devPath": "http://localhost:8080/",
+    "beforeBuildCommand": "cd src-vue && npm run build && cp dist/* ../dist -r",
+    "beforeDevCommand": "cd src-vue && npm run dev",
+    "devPath": "http://localhost:5173/",
     "distDir": "../dist"
```

Comment:
Just noticed, instead of moving vue project to `../dist/` we could also just point `distDir` to `../src-vue/dist/` ^^

Reply:
Yep, that might be included in the future PR removing old UI

Reply:
https://github.com/Alystrasz/FlightCore/pull/10